core-kotlin-modules/core-kotlin-strings-3/src/test/kotlin/com/baeldung/string_extension_functions/StringExtensionFunctionClassUnitTest.kt

```diff
@@ -0,0 +1,89 @@
+package com.baeldung.string_extension_functions
+
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+internal class StringExtensionFunctionClassUnitTest {
+
+
+    @Test
+    fun `using replace() method`(){
+        val inputString0 = "Jelly"
+        val inputString1 = "Kotlin Replace Program"
+        assertEquals("Jezzy", inputString0.replace( 'l', 'z'))
+        assertEquals("Pelly", inputString0.replace('j', 'P', true))
+        assertEquals("Jelly",inputString0.replace('j', 'P'))
+        assertEquals("Kotlin Replace Examples", inputString1.replace("PROGRAM", "Examples", true))
+    }
+
+    @Test
+    fun `using uppercase() method`(){
+        var str = "Extension Functions"
+        var result = str.uppercase()
+        assertEquals("EXTENSION FUNCTIONS",result)
+    }
+
+    @Test
+    fun `using lowercase() method`(){
+        var str = "Extension Functions"
+        var result = str.lowercase()
+        assertEquals(result, "extension functions")
```

Comment:
need to swap the args

Reply:
That has been fixed now.
Thank you for pointing it out.